src/wallet.c

```diff
@@ -215,13 +222,23 @@ int wallet_generate_key(HDNode *node, const char *keypath, int keypath_len,
     memcpy(node->private_key, privkeymaster, 32);
     hdnode_fill_public_key(node);
 
-    pch = strtok(kp, " /,m\\");
+    pch = strtok(kp + 2, delim);
```

Comment:
If I'm right, `kp` does not get a length check, so `kp+2` could touch random memory?